mozilla_schema_generator/glean_ping.py

```diff
@@ -109,7 +109,7 @@ def get_dependencies(self):
                 dependencies.append(repos_by_dependency_name[name])
 
         if len(dependencies) == 0:
-            logging.info(f"For {self.repo_name}, using default Glean dependencies")
+            logging.info(f"No dependencies found for {self.repo_name}")
             return self.default_dependencies
```

Comment:
Given `self.default_dependencies` is empty anyway now, this whole check can go. We return the empty `dependencies` a few lines further down anyway.

Reply:
👍